lib/stdlib/src/edlin_expand.erl

```diff
@@ -20,43 +20,780 @@
 -module(edlin_expand).
 
 %% a default expand function for edlin, expanding modules and functions
-
--export([expand/1, format_matches/1]).
+-include("../../kernel/include/eep48.hrl").
+-export([expand/1, expand/3, format_matches/1, get_exports/1]).
 
 -import(lists, [reverse/1, prefix/2]).
 
-%% expand(CurrentBefore) ->
-%%	{yes, Expansion, Matches} | {no, Matches}
+%% Will remove this before delivery
+%% Questions:
+%% Types:
+%%  should we print all the leaf concrete types or stop at user_type and remote_type?
+%%   - file:open("/", )
```

Comment:
I wonder if maybe it would be a good idea to expand one "level" of remote types per <TAB> you hit. I.e. in the example of `file:open("/", )` we first show:

```
directory    mode()     ram
```
then expand mode() to
```
directory read write append exclusive raw binary {delayed_write, non_neg_integer(), non_neg_integer()} delayed_write {read_ahead, pos_integer()} read_ahead compressed {encoding, unicode:encoding()} sync ram
```
It should however not expand the encoding as that is within another compound element, so leave that until the user does:

```
file:open("/",[{encoding, <TAB><TAB>
```

Reply:
Just added this handling, I like it